js/templates/modals/editListing/editListing.html

```diff
@@ -150,7 +156,7 @@ <h2 class="h4 clrT"><%= ob.polyT('editListing.sectionNames.photos') %></h2>
       </section>
 
       <section class="js-scrollToSection js-sectionShipping shippingSection <% if (ob.listing.metadata.contractType !== 'PHYSICAL_GOOD') print('disabled') %> <% if (ob.listing.shippingOptions.length) print('hasShippingOptions') %>">
-        <div class="pad overflowAuto emptyShippingOptionsSection contentBox padMd clrP clrBr clrSh3 tx3">
+<!--         <div class="pad overflowAuto emptyShippingOptionsSection contentBox padMd clrP clrBr clrSh3 tx3">
```

Comment:
@rmisio @jjeffryes I commented this section out (pictured below). We no longer need it since we have the **Shipping Option 1** placeholder. Is this going to introduce any issues? Can I completely remove it from the code? 
![screenshot 2017-01-04 13 56 03](https://cloud.githubusercontent.com/assets/13066886/21660751/95f85496-d285-11e6-8d43-3b523a8ad77f.png)

This still exists
![screenshot 2017-01-04 13 53 34](https://cloud.githubusercontent.com/assets/13066886/21660712/67091f4e-d285-11e6-86df-d54b5571aafe.png)